engine/src/cubos/engine/assets/assets.cpp

```diff
@@ -600,4 +605,18 @@ std::vector<AnyAsset> Assets::listAll() const
         out.emplace_back(entry);
     }
     return out;
+}
+
+std::type_index Assets::type(const AnyAsset& handle) const
+{
+    auto assetEntry = this->entry(handle);
+    CUBOS_ASSERT(assetEntry != nullptr, "Could not find asset's type: No entry found!");
```

Comment:
```suggestion
    CUBOS_ASSERT(assetEntry != nullptr, "Could not find asset's type");
```

Turns out `entry()` already logs the error reason anyway